docs/ref/release_notes.rst

```diff
@@ -19,6 +19,12 @@ Features
 * Added saving for data on irregular, non-rotated grids (grid definition
   template 4 in template table 3.1)
 
+  .. note::
+
+      Loading for the Mercator projection is only available using iris
```

Comment:
I think this applies to save also, in fact : It will try to read the 'standard_parallel' of the coord system, which only exists from Iris 2.1
( see [here](https://github.com/corinnebosley/iris-grib/blob/f865b0e97ef464317325172c2a4110ea90d55df9/iris_grib/_save_rules.py#L508) )